src/components/Header.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import logo from '../Images/Logo-1.png';
+import '../css/Header.css';
+
+export default function Header() {
+  return (
+    <header>
+      <div className="branding">
+        <img src={logo} alt="" />
+        <h1>Space Traveler&apos;s Hub</h1>
+      </div>
+      <nav>
+        <ul>
+          <li>
+            <a href="/" className="menu-item">Rockets</a>
+          </li>
+          <li>
+            <a href="/" className="menu-item">Missions</a>
+          </li>
+          <li>
+            <a href="/" className="menu-item">My Profile</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
```

Comment:
Nice work so far, you are expected to use NavLink for the navigation according to the instruction, Add the NavLink in the Navbar.js and import it into Header.js component